src/main.rs

```diff
@@ -135,6 +139,16 @@ fn bash(imports_path: Option<&&str>) {
     print!("{}", TABRY_BASH_SH);
 }
 
+const TABRY_FISH_SH: &str = include_str!("../shell/tabry_fish.fish");
+fn fish(imports_path: Option<&&str>) {
+    if let Some(path) = imports_path {
+        println!("set -x _tabry_rs_imports_path '{}'", escape(path));
```

Comment:
Does it work without exporting? (The vars for the bash version aren't, because the completion function runs in the same process).

Reply:
Yeah, I now realize that `TABRY_IMPORT_PATH` is the variable passed to `tabry`

Reply:
Update: it doesn't work without exporting, and I can't figure out how to pass it to the completion function. It should only pass it to `tabry`, but I don't think `tabry` cares at all?

Reply:
Not sure what you don't mean "tabry cares at all"?
Exporting is fine but I wonder if you can just set it as TABRY_IMPORT_PATH here?